src/main.rs

```diff
@@ -13,6 +14,18 @@ use rust_embed::RustEmbed;
 mod models;
 mod routes;
 
+const BINARY_VERSION: &str =
+    concat!(env!("CARGO_PKG_VERSION"), env!("GIT_HASH"));
+const SERVER_VERSION: &str = concat!(
+    "bin v.",
+    env!("CARGO_PKG_VERSION"),
+    " (",
+    env!("GIT_HASH"),
+    ") (Rocket)"
+);
+static BINARY_ETAG: Lazy<String> =
+    Lazy::new(|| sha256::digest(BINARY_VERSION));
```

Comment:
I went on searching for a library which provides a constant function to generate sha256 digests. 
The closest I got was https://github.com/saleemrashid/sha2-const, but it brings in more weight than we would have after replacing it with both once-cell and sha256.